src/Microsoft.Health.Fhir.Tests.Common/NotTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Xunit;
+
+namespace Microsoft.Health.Extensions.Xunit
```

Comment:
Should this namespace match the project the file is in?

Reply:
fixed